provider-messaging/src/components/organisms/c-message-list.jsx

```diff
@@ -83,21 +100,22 @@ const CMessageList = () => {
 
   return (
     // TODO!
-    <div className="c-message-list px-9 w-full lg:px-12 relative lg:h-screen table lg:flex lg:flex-col justify-between overflow-x-hidden">
-      <div>
-        <MChatTools
-          setDiscussion={setDiscussion}
-          selectedDiscussion={discussion}
-        />
-        {discussion?.messages?.map((message) => {
+    <div className="c-message-list w-full relative lg:h-screen table lg:flex lg:flex-col justify-between overflow-x-hidden">
+      <MChatTools
+        setConversation={ setConversation }
```

Comment:
Thanks for re-wording the variable to match the domain language (discussion -> conversation) :)